docs/products/mysql/howto/max-number-of-connections.rst

```diff
@@ -0,0 +1,29 @@
+Calculate the maximum number of connections for MySQL
+=====================================================
+
+
+The number of simultaneous connections in Aiven for MySQL depends on the usable memory on the server. The usable memory is the total memory on the node minus the operating system and management overhead. This overhead is currently estimated as 350 MiB (0.341797 GiB). 
```

Comment:
Is it useful to go to six digits after the decimal point?
```
>>> 350.0 / 1024
0.341796875
```
so that's already rounding a bit. Not sure how much precision is needed, but is `approximately 0.34` good enough?

Reply:
I think an approximation is enough. In the end, it will be rounded to the nearest integer. I also agree that some examples would be good, so I have added. It is not clear why the changes were done for those two plans but the explanation is that plans with +4Gib have less memory-constrained compared to the other one.

Reply:
👍